docs/UserGuide.md

```diff
@@ -361,7 +361,19 @@ Format: `find KEYWORDS`
 
 Examples:
 * `find discuss user guide John` returns a meeting with title `discuss user guide` and another with description `with Mr John`.
-* `find Mcd0nalds` returns a meeting with title `Meet Mcd0nalds` and another with description `Cater Mcd0nalds breakfast`.
+* `find Mcdonalds` returns a meeting with title `Meet Mcdonalds` and another with description `Cater Mcdonalds breakfast`.
+
+#### Listing all meetings: `list`
+
+Lists all meetings from your meeting list. 
```

Comment:
Can we stick to "Shows a list of all meetings in your meeting list" for consistency? (or is this too repetitive?)